interpreter/cling/lib/Interpreter/ValuePrinter.cpp

```diff
@@ -549,7 +549,7 @@ static const char* BuildAndEmitVPWrapperBody(cling::Interpreter &Interp,
                                              clang::QualType QT,
                                              const void* ValPtr)
 {
-  const clang::SourceLocation noSrcLoc;
+  const clang::SourceLocation noSrcLoc = {};
```

Comment:
Why is this needed?

Reply:
It looks like very old compilers like Clang 3.8 do not support default initialization of a `const` object, without an explicitly specified default constructor - https://github.com/reikdas/cling/runs/2227072695?check_suite_focus=true#step:6:2897

Reply:
Clang 3.8 is historical. Can we just not support that one, please?

Reply:
That'd mean dropping a build that we supported. If that's the only change why not keeping it and the Ubu16 platform?

Reply:
Because any single change is "the only change" and we will accumulate "single changes" at the cost of baroque code. clang 3.8 is dead old and this is a compiler bug; let's drop support. Coverage matters a ton when it comes to GCC 10, llvm 12 etc.

Reply:
Updated the PR to drop Ubuntu 16.04 Clang build, and revert this change.